application/F3DStarter.cxx

```diff
@@ -713,12 +713,17 @@ int F3DStarter::Start(int argc, char** argv)
   // but this duplicate the initialization value as it is present in
   // F3DOptionTools::DefaultAppOptions too
   bool dryRun = false;
+  bool noRender = false;
+  if (cliOptionsDict.find("dry-run") != cliOptionsDict.end())
+  {
+    dryRun = f3d::options::parse<bool>(cliOptionsDict["dry-run"]);
+  }
   if (cliOptionsDict.find("no-render") != cliOptionsDict.end())
   {
-    dryRun = f3d::options::parse<bool>(cliOptionsDict["no-render"]);
+    noRender = f3d::options::parse<bool>(cliOptionsDict["no-render"]);
   }
```

Comment:
actually we do not need to check for `no-render` here at all. The fix is "just" to replace `"no-render"` by `"dry-run"`

Reply:
> actually we do not need to check for `no-render` here at all. The fix is "just" to replace `"no-render"` by `"dry-run"`

and for this case:
```
f3d --config=config_build --no-render
```
We doesn't want to read the config file?

Reply:
> f3d --config=config_build --no-render

In that cas we want to read the config file named "config_build", `no-render` has nothing to do with config files.

Reply:
From `f3d --help`:
```
[...]
--dry-run [=<bool>(=1)]
  Do not read the configuration file
  (default: false)
--no-render [=<bool>(=1)]
  Do not read the configuration file
  (default: false)
[...]
```
If it is writen the same line, it should not read the same configuration file?
Perhaps should we change the option tool too?
```patch
 application/F3DOptionsTools.cxx | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/application/F3DOptionsTools.cxx b/application/F3DOptionsTools.cxx
index 31a70bbf..acdcf5ea 100644
--- a/application/F3DOptionsTools.cxx
+++ b/application/F3DOptionsTools.cxx
@@ -67,7 +67,7 @@ static inline const std::array<CLIGroup, 8> CLIOptions = {{
       { "readers-list", "", "Print the list of readers", "", "" },
       { "config", "", "Specify the configuration file to use. absolute/relative path or filename/filestem to search in configuration file locations", "<filePath/filename/fileStem>", "" },
       { "dry-run", "", "Do not read the configuration file", "<bool>", "1" },
-      { "no-render", "", "Do not read the configuration file", "<bool>", "1" },
+      { "no-render", "", "Do not render", "<bool>", "1" },
       { "rendering-backend", "", "Backend to use when rendering (auto|glx|wgl|egl|osmesa)", "<string>", "" },
       { "max-size", "", "Maximum size in Mib of a file to load, negative value means unlimited", "<size in Mib>", "" },
       { "watch", "", "Watch current file and automatically reload it whenever it is modified on disk", "<bool>", "1" },
```

Reply:
I suggest that:
* 7b6e404e Starter: remove no-render
* 9966520c OptionsTools: change no-render description

Reply:
Indeed, its looking good!

By the way, in the issue https://github.com/f3d-app/f3d/issues/1637 @snoyer suggest to rename the `--dry-run` option to `--no-config`, do you think you can do it ?

Then you would just need to add a test for it.

Reply:
Normally the renaming should be good (in commit comment, there are my commands applied)
* 04f20b15 rename dryRun by noConfig
* 94d3123e rename dry-run by no-config

Reply:
Nice, you may also need to update the markdown doc, in `doc` directory.

Reply:
done: 0cf02b21d05f8a3ffcf3474bee1bfed112223fa4